pkg/checker/test_script/cleanup_test_environment.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Hi @NishantBansal2003 😄 

You can move this part out of the `checker` package and into a new package called `mock`, where you can wrap the process of creating a virtual OCI environment for testing and make it available to all test cases, not just the `checker` package.